### PR TITLE
INVGOV-9112 IB-2954 post likes and comments sort

### PR DESCRIPTION
**Pull request by ParryLuke**

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -1332,6 +1332,49 @@ paths:
...
         required: true
+  /posts/list:
+    get:
+      summary: List of posts.
+      tags:
+        - posts
+      parameters:
+        - description: Field to sort by
+          in: query
+          name: sort
```
Comment (jmorahan) [inline api.swagger.yaml]: No default and not required - so what do we sort by if it is omitted?

File: api.swagger.yaml [hunk 2]
```diff
@@ -1332,6 +1332,49 @@ paths:
...
+          schema:
+            type: string
+            enum:
+              - comment_count
+              - likes_count
```
Comment (jmorahan) [inline api.swagger.yaml]: why is comment singular and likes plural?

Comment (ParryLuke) [reply api.swagger.yaml]: This is for consistency with the existing post-schema

File: api.swagger.yaml (hunk 1 above)
Comment (ParryLuke) [reply api.swagger.yaml]: update

**Pull request merged**